.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,52 @@
+name: Scorecard supply-chain security
+on:
+  branch_protection_rule:
+  schedule:
+    - cron: '01 12 * * 1'
```

Comment:
can you add a comment what schedule does it represent

Reply:
done